### PR TITLE
Update description in readme

### PR DESCRIPTION
**Pull request by moarychan**
- Remove unnecessary descriptions
- Maintain code and documentation consistency.

### CODE REVIEW

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server-obo/README.md [hunk 1]
```diff
@@ -21,31 +21,8 @@ Another sample [azure-spring-boot-sample-active-directory-resource-server] will
...
-    <dependency>
-      <groupId>org.springframework.boot</groupId>
-      <artifactId>spring-boot-starter-web</artifactId>
-    </dependency>
-</dependencies>
-```
-
 ### Configure your middle-tier Web API A
 1. Expose two scopes for ***Web API A***, `Obo.Graph.Read` and `Obo.File.Read`. The **Application ID URI** is generated by default.  
-   **NOTE**: Theses scopes (`Obo.Graph.Read`, `Obo.File.Read`) should be added to the **Web APP**.
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server-obo/README.md:48]: Why are we removing this?

Comment (moarychan) [reply sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server-obo/README.md:48]: For simplicity, remoted. Current scopes of the resource server can be added by Web App or Resource Server, not just Web App.

**Review by chenrujun — COMMENTED** [review]
LGTM about deleting dependencies in readme.

**Review by saragluna — APPROVED** [review]
LGTM

**Pull request merged**